pkg/controller/attach_detach.go

```diff
@@ -70,6 +82,7 @@ type ReconcileAttachDetach struct {
 	cloudDiskAttacher CloudDiskAttachDetacher
 	stateLock         *sync.Map
 	retryInfo         *retryInfo
+	volumeInfos       *sync.Map
```

Comment:
If this is only used during `Recover`, do we really need to use `sync.Map`? It doesn't seem to be accessed by multiple goroutines, so it seems like overkill.